swf/src/write.rs

```diff
@@ -2475,6 +2529,15 @@ impl<W: Write> Writer<W> {
         Ok(())
     }
 
+    fn write_tag_code_and_length_force_long(&mut self, tag_code: u16, length: u32) -> Result<()> {
+        // TODO: Test for tag code/length overflow.
+        let mut tag_code_and_length = tag_code << 6;
+        tag_code_and_length |= 0b111111;
+        self.write_u16(tag_code_and_length)?;
+        self.write_u32(length)?;
+        Ok(())
+    }
```

Comment:
Although the Flash IDE tends to export these with long tag headers, the Flash Player seems to handle short tag lengths here fine. The SWF spec also does not state that these needs to be long tag header.

Reply:
I have experienced segfault with some .swf files after being rewritten with short tag